tests/serial/watch-integral/WatchIntegralScaleAndNoScale.xml

```diff
@@ -26,7 +24,6 @@
 
     <participant name="SolverTwo">
       <provide-mesh name="MeshTwo" />
-      <read-data name="DataTwo" mesh="MeshTwo" />
```

Comment:
I guess this needs to be `DataOne`, right? Otherwise, which data are you actually 'watching' here (as there is also no write data tag.

Reply:
Not sure. I don't know what this test is supposed to be doing. I just stumbled over the config and found it odd. I currently don't have time to play around with this test and try to thoroughly understand it. Feel free to play around with it, if you want. Maybe some input from @fsimonis?

Reply:
Should probably be `DataOne`, yes.

Reply:
Fixed in [f172994](https://github.com/precice/precice/pull/1526/commits/f17299490b3a2979595f9459c3cf870082bb1a35)